1001-2000/LIT1259Daniel.xml

```diff
@@ -35,9 +35,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <abstract>
-            <p>Generic ID for the biblical Book of Daniel, referring to a recension that does not include the stories of Susanna, Bel and the 
-               Dragon, the Prayer of Azariah and the Three Holy Children, which belong to the so-called deuterocanonical books, or to a or to a 
-               text for which this is unknown.</p>
+            <p>General ID for the biblical <ref type="work" corresp="wd:Q80115">Book of Daniel</ref>, referring to a recension that does 
```

Comment:
judging by the wikidata structure, ID q80115 stands for `3529`, not for `1259`

**Q218087 and Q16930210 have the part of relation to q80115** 

to avoid unnecessary confusion, unless you are 100% clear about it, remove wd.

now you have inserted in the abstract the same wikidata id for both 1259 and 3529